dlp/deid.py

```diff
@@ -83,8 +83,72 @@ def deidentify_with_mask(
 
 # [END dlp_deidentify_masking]
 
+# [START dlp_deidentify_replace]
+def deidentify_with_replace(
+    project,
+    string,
```

Comment:
```suggestion
    item,
```
For consistency with the existing function

Reply:
Done.